test/org/sagebionetworks/bridge/services/ScheduledActivityServiceRecurringTest.java

```diff
@@ -130,7 +127,7 @@ public void retrievalActivitiesAcrossTimeAndTimeZones() throws Exception {
         String pst1 = DateTime.now(PST).toLocalDate().toString();
         String pst2 = DateTime.now(PST).plusDays(1).toLocalDate().toString();
         String pst3 = DateTime.now(PST).plusDays(2).toLocalDate().toString();
-        String pst4 = DateTime.now(PST).plusDays(3).toLocalDate().toString();
+        //String pst4 = DateTime.now(PST).plusDays(3).toLocalDate().toString();
```

Comment:
It shouldn't, this is being reverted back to have pst4, I wonder if this wouldn't break this test, it only passes with all four PST dates right now and I don't believe I did anything today that would have caused a failing test to succeed.